Content.Server/Chemistry/Components/SolutionInjectOnCollideComponent.cs

```diff
@@ -23,5 +23,11 @@ internal sealed class SolutionInjectOnCollideComponent : Component
 
         [DataField("transferEfficiency")]
         private float _transferEfficiency = 1f;
+
+        [DataField("canPenetrateArmor")]
+        public bool? CanPenetrateArmor { get; }
+
+        [DataField("canPenetrateHelmet")]
+        public bool? CanPenetrateHelmet { get; }
```

Comment:
Why are these nullable, and also why aren't they fields?

Also shouldn't it check for mask slot?

Reply:
yeah they aren't meant to be nullable , I fixed it